HuggingFaceTest.scala

```diff
@@ -0,0 +1,54 @@
+import org.llm4s.imagegeneration._
```

Comment:
This file should not live in the root directory - not sure if it should be in the build at all.